index.html

```diff
@@ -972,12 +973,14 @@
       <li>
         Writing <a>Web NFC content</a> to an <a>NFC tag</a> does not need to
         <a>obtain permission</a>, if the <a>NFC tag</a> contains
-        <a>trusted integrity NFC content</a> and the value of the <a>NDEF Id</a>
-        field is equal to the <a>ASCII serialized origin</a> of the
-        <a>incumbent settings object</a>.
+        <a>trusted integrity NFC content</a> which is a <a>Web NFC message</a>
+        and the <a>origin</a> part of the <a>Web NFC Id</a> field is equal to
+        the <a>ASCII serialized origin</a> of the
+         <a>incumbent settings object</a>.
         Otherwise the UA must <a>obtain permission</a> for
         sending <a>NFC content</a> which overwrites existing information.
-        See also the <a>Writing or pushing content</a> section.
+        See also the steps to <a>create a Web NFC record</a> in the
+        <a>Writing or pushing content</a> section.
```

Comment:
You can also use [`<a href="#section-id"></a>`](http://www.w3.org/respec/ref.html#section) to link to sections.